.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
do not checkin .vscode folder
it's good to be mindful of one's teammates and not force preferences on them unnecessarily